Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+FROM alpine:3.13.4
+
+ARG ARCH=amd64
+
+ARG KUBERNETES_VERSION=1.17.3
+ARG REST_SERVER_VERSION=0.10.0
+
+RUN apk add --no-cache curl && \
+    cd /usr/local/bin && \
+    curl -sLO https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VERSION}/bin/linux/${ARCH}/kubectl && \
```

Comment:
you could use something like to grab `ARCH` https://github.com/eclipse-che/che-plugin-registry/blob/b5caed539d9faf117d4b8b73eabb8198653af9b1/sidecars/kubernetes-tooling/Dockerfile#L20-L22